src/components/utils/sendMessageBackground.tsx

```diff
@@ -17,6 +17,12 @@ export async function sendAccountUpdatedMessage(selectedAccount: AccountData | n
       }
     );
   });
+  chrome.runtime.sendMessage({
+    type: 'SELECTED_ACCOUNT_UPDATED',
+    data: {
```

Comment:
Should be `data: selectedAccount`, handler was updated https://github.com/0xSpaceShard/starknet-rivet/pull/65/commits/89547a868f7dd8f213e9b2480b9d17eb6d929258

Reply:
done